README.md

```diff
@@ -1,21 +1,35 @@
 # Pharmacy Information Management System
 
-### Setup 
+## Setup
 
+The application is divided into a frontend and backend service. For an initial setup you will need to run `npm install` in both the frontend and backend subdirectories. This will install the dependencies required for each of the services.
```

Comment:
We can probably drop "you will need to" and "can be done by" from the various lines in the readme. Instructional text reads better when it's more direct.